### ISSUE TITLE
Symfony 5 Bundle Route Not loaded

### ISSUE THREAD

**Issue by gaet592010** [GregoireHebert/docusign-bundle#93]
Hi ! 

I tried to implement this bundle in my Symfony 5 application, but the bundle's routes seem to doesn't loaded 

So when i follow instruction, like going to the route `/docusign/consent/default` the route doesn't exist.

I tried to clear cache .

**Comment by vincentchalamon** [comment]
Hi @gaet592010,

Thanks for your feedback. Indeed, this point is missing in the recipe and the documentation, we're currently fixing it. Meanwhile, you can manually declare the bundle routes using the following syntax:
```yaml
# config/routes/docusign.yaml
docusign:
    resource: .
    type: docusign
```

**Comment by GregoireHebert** [comment]
You are totally right ! I forgot to add this part in the doc xD

My apologies for the inconvenience.

**Comment by gaet592010** [comment]
Thank you !

It's working  well !!!

**Comment by GregoireHebert** [comment]
Thanks for your involvement @vincentchalamon and @gaet592010 :D 
v5 has been updated, merged and tagged 5.3.5 !
